tests/PhpInfoDataNormalizerTest.php

```
@@ -389,6 +818,29 @@ public function testFromOutputShortenPathsAgainstHomeDirectory(): void
+
+        self::assertStringContainsString(
+            "'--prefix=~/.local/php'",
+            $view->configureCommand,
+            'Account name must not leak through build flags.',
+        );
+        self::assertStringContainsString(
+            "'--with-config=/opt/home/dev/etc'",
+            $view->configureCommand,
+            'A directory merely ending in the home path must survive.',
+        );
+    }
```

PHPMD (2.15.0)

[error] testFromOutputShortensHomeDirectoryInsideConfigureCommand accesses the super-global variable $_SERVER. (undefined)

(Superglobals)